torch/csrc/jit/passes/python_print.cpp

```diff
@@ -949,6 +954,10 @@ struct PythonPrintPass {
         stmt << "uninitialized(" << node->output()->type()->python_str() << ")";
       } break;
       case prim::Constant: {
+        if (node->outputs().size() == 1 &&
```

Comment:
It is very confusing to have a function printRHS that always prints, except for function constants. Can this be refactored? It will trip up someone trying to understand this code.